eppo/build.gradle

```diff
@@ -63,17 +63,16 @@ ext.versions = [
     "androidx_runner": "1.5.2",
     "gson": "2.9.1",
     "okhttp": "4.10.0",
-    "wiremock": "2.34.0"
+    "commonsio": "2.14.0"
```

Comment:
We were using some classes, in apache commons, that came with Wiremock outside of our mock server.